app/assets/javascripts/discourse/components/group-card-contents.js.es6

```diff
@@ -23,6 +23,9 @@ export default Ember.Component.extend(CardContentsBase, CleansUp, {
   @computed('group.user_count', 'group.members.length')
   moreMembersCount: (memberCount, maxMemberDisplay) => memberCount - maxMemberDisplay,
 
+  @computed('group.name')
+  groupClass: (name) => name ? `group-card-${name}` : '',
```

Comment:
Is there a case where `group.name` or `username` would be `null`?

Reply:
From my testing, username gets set to null and group.name is undefined when the card is hidden

Reply:
:+1: